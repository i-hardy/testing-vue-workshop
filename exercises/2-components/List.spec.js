import List from "./List.vue";

test('renders "no items" if items prop is undefined', () => {});

test("renders text using each item in items prop array");
