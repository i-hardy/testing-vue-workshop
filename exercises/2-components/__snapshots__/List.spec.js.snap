// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders text using each item in items prop array 1`] = `
<ul>
  <li>
    
    red
  
  </li>
  <li>
    
    blue
  
  </li>
  <li>
    
    green
  
  </li>
</ul>
`;
